test/lib/ufe/testParentCmd.py

```diff
@@ -997,43 +997,40 @@ def testUnparentMultiStage(self):
-            proxyShapeItem1 = ufe.Hierarchy.createItem(ufe.PathString.path(
+            stage2ParentItem = ufe.Hierarchy.createItem(
+                ufe.PathString.path('|simpleSceneUSD_TRS_mayaUsdProxy1|simpleSceneUSD_TRS_mayaUsdProxyShape1,/cylinderXform'))
+            stage1ProxyShapeItem = ufe.Hierarchy.createItem(ufe.PathString.path(
                 '|mayaUsdProxy1|mayaUsdProxyShape1'))
-            proxyShapeItem2 = ufe.Hierarchy.createItem(ufe.PathString.path(
+            stage2ProxyShapeItem = ufe.Hierarchy.createItem(ufe.PathString.path(
                 '|simpleSceneUSD_TRS_mayaUsdProxy1|simpleSceneUSD_TRS_mayaUsdProxyShape1'))
-            cylinder1 = ufe.Hierarchy.hierarchy(cylinderItem1)
-            shadingGroup2 = ufe.Hierarchy.hierarchy(shadingGroupItem2)
-            proxyShape1 = ufe.Hierarchy.hierarchy(proxyShapeItem1)
-            proxyShape2 = ufe.Hierarchy.hierarchy(proxyShapeItem2)
+            stage1ParentHierarchy = ufe.Hierarchy.hierarchy(stage1ParentItem)
+            stage2ParentHierarchy = ufe.Hierarchy.hierarchy(stage2ParentItem)
+            stage1ProxyShapeHierarchy = ufe.Hierarchy.hierarchy(stage1ProxyShapeItem)
+            stage2ProxyShapeHierarchy = ufe.Hierarchy.hierarchy(stage2ProxyShapeItem)
 
             def checkUnparent(done):
-                proxyShape1Children = proxyShape1.children()
-                proxyShape2Children = proxyShape2.children()
-                cylinder1Children = cylinder1.children()
-                shadingGroup2Children = shadingGroup2.children()
+                stage1ProxyShapeChildren = stage1ProxyShapeHierarchy.children()
+                stage2ProxyShapeChildren = stage2ProxyShapeHierarchy.children()
+                stage1ParentChildren = stage1ParentHierarchy.children()
+                stage2ParentChildren = stage2ParentHierarchy.children()
                 self.assertEqual(
-                    'pCube1' in childrenNames(proxyShape1Children), done)
+                    'pCube1' in childrenNames(stage1ProxyShapeChildren), done)
                 self.assertEqual(
-                    'pCube1' in childrenNames(cylinder1Children), not done)
+                    'pCube1' in childrenNames(stage1ParentChildren), not done)
                 self.assertEqual(
-                    'initialShadingGroup_lambert' in childrenNames(proxyShape2Children), done)
+                    'pCylinder1' in childrenNames(stage2ProxyShapeChildren), done)
                 self.assertEqual(
-                    'initialShadingGroup_lambert' in childrenNames(shadingGroup2Children), not done)
+                    'pCylinder1' in childrenNames(stage2ParentChildren), not done)
 
             checkUnparent(done=False)
 
-            # Use relative parenting, else trying to keep absolute world
-            # position of Lambert node fails (of course).
-            cmds.parent(cubePathStr1, lambertPathStr2, w=True, r=True)
+            cmds.parent(stage1ItemPath, stage2ItemPath, w=True)
```

Comment:
Previously, this call was unparenting a Shader. Unparenting or reparenting a Shader seems to fail in Maya2022 for some reason unless `relative=ture`. I think that's the only reason why the `relative` flag was set in the first place.

Since we are not unparenting a Shader anymore, the `relative` flag in is no longer necessary. If we leave it in, I feel like a reader might wonder, why it is set, so I felt like removing it would make the code more readable.

Reply:
Okay that explains it. Thanks.